handler.ts

```diff
@@ -157,56 +90,48 @@ export const makeGeocoderRequests = async (
   delete peliasQSP.layers
 
   // Run both requests in parallel
-  let [primaryResponse, customResponse]: [
-    FeatureCollection,
-    FeatureCollection
-  ] = await Promise.all([
-    cachedGeocoderRequest(
-      getPrimaryGeocoder(),
-      apiMethod,
-      convertQSPToGeocoderArgs(event.queryStringParameters),
-      // @ts-expect-error Redis Typescript types are not friendly
-      redis
-    ),
-    // Custom request is either through geocoder package or "old" pelias method
-    getTransitGeocoder()
-      ? cachedGeocoderRequest(
-          getTransitGeocoder(),
-          'autocomplete',
-          convertQSPToGeocoderArgs(event.queryStringParameters),
-          null
+  let responses: FeatureCollection[] = await Promise.all(
+    geocoders.map((geocoder) =>
+      cachedGeocoderRequest(getGeocoder(geocoder), apiMethod, {
+        ...convertQSPToGeocoderArgs(event.queryStringParameters),
+        items: pois
+      })
+    )
+  )
+
+  responses = await Promise.all(
+    responses.map(async (response, index) => {
+      // If backup geocoder is present, and the returned results are garbage, use the backup geocoder
+      if (
+        backupGeocoders[index] &&
+        !checkIfResultsAreSatisfactory(
+          response,
+          event.queryStringParameters.text
         )
-      : fetchPelias(
-          TRANSIT_BASE_URL,
-          apiMethod,
-          `${new URLSearchParams(peliasQSP).toString()}&sources=transit${
-            CSV_ENABLED && CSV_ENABLED === 'true' ? ',pelias' : ''
-          }`
+      ) {
+        const backupGeocoder = getGeocoder(backupGeocoders[index])
+        console.log('backup geocoder used!')
+        return await backupGeocoder[apiMethod](
```

Comment:
Probably need to keep the comment on why backup geocoder responses aren't cached.

Reply:
Nothing at all is cached!

Reply:
That was mentioned in deleted line 189. Can we keep a similar mention?